app/components/ui/files/registration-modal/component.js

```diff
@@ -174,10 +188,17 @@ export default Ember.Component.extend({
             Ember.$('#results').removeClass('hidden');
```

Comment:
I also avoid using jQuery as much as possible. I favor having variables and rendering the handlebars/html-bars sections using these variables, in this case, something like:

```
    {{#if resultsVisible}}
        <div id="results"> 
            ...
        </div>
    {{/if}}
```

Reply:
I agree - there are a number of additional handlebars rules we have included in WholeTale using the ember truth helpers too that allow you to do quite complex logic in the templates directly: https://github.com/jmurphyau/ember-truth-helpers

Reply:
@ThomasThelen, where are we on this? Can I merge and close this PR already?

Reply:
Ready to go!